frontend/src/Search/AddNewItem.js

```diff
@@ -166,7 +166,9 @@ class AddNewItem extends Component {
           {
             !isFetching && !error && !items.length && !!term &&
               <div className={styles.message}>
-                <div className={styles.noResults}>Couldn't find any results for '{term}'</div>
+                <div className={styles.noResults}>
+                  {translate('CouldntFindAnyResultsForTerm', [term])}
+                </div>
                 <div>
                   You can also search using the
```

Comment:
How about this?

Reply:
updated & pushed for both

Note: dropped the links to MB - didn't seem too useful IMO